### DISCUSSION TITLE
Ignore the arguments if it is Any type

### DISCUSSION THREAD

**Discussion by Kither12** [microsoft/pyright#9493] [General]
In Python, the `Any` type from the `typing` module is used to indicate that a value can be of any type. This includes `None`. By allowing arguments of type `Any` to be ignored, we can streamline function definitions and reduce unnecessary complexity.

**Comment by erictraut** [comment]
I'm not sure what you mean. Pyright is a standards-based type checker, and most of its behaviors regarding `Any` are dictated by the [Python typing spec](https://typing.readthedocs.io/en/latest/spec/special-types.html#any). If you think that pyright is deviating from the spec or is otherwise behaving in a way that you don't expect, could you provide a self-contained minimal code sample that demonstrates the problem you're seeing?
